rddl_info/crawl.py

```diff
@@ -62,8 +64,12 @@ def download_obj(url: str):
 
 def write_storage_entry(tx: dict, obj: dict, storage):
     try:
+        for rddl_node in rddl_node_list:
+            if rddl_node["pub"] == tx["inputs"][0]["owners_before"][0]:
+                o = urlparse(rddl_node["uri"])
+                break
         data = [
-            tx["inputs"][0]["owners_before"][0],
+            o.netloc,
```

Comment:
what if the pub key wasn't within the validators? Then we will have an empty o. and it will fail of have bad data. we should not notarize or save/store the data in that case.